tests/Unit/Transport/GrpcTransportTest.php

```diff
@@ -301,6 +301,7 @@ public function testBidiOpeningRequestLogsRpcName()
 
         $this->assertNotEmpty($unserializedBuffer);
         $this->assertNotEmpty($unserializedBuffer['rpcName']);
+        $this->assertEquals($unserializedBuffer['rpcName'], $rpcName);
```

Comment:
one final nit but, in PHPUnit, the expected value goes first. So this should be 
```suggestion
        $this->assertEquals($rpcName, $unserializedBuffer['rpcName']);
```

This can be important for the error output, because it will say "Expected string Foo does not match expected Bar", which can be important when debugging (I've had the order being wrong mess me up before)

Reply:
Damn it! I always make this mistake!